src/main/java/Duke/DukeCommands.java

```diff
@@ -0,0 +1,71 @@
+package Duke;
+
+import java.util.ArrayList;
+
+public class DukeCommands {
+    public static String spacer = "__________________________________________\n";
+    public static ArrayList<DukeTasks> list = new ArrayList<>();
+    static Integer taskLength = 0;
+
+    // Still requires addition of speach lines
+
+    /**
+     * Adds a task with the specified name to a list, and prints a response to the user. 
+     * 
+     * @param taskName Name of task added
+     */
+    public static void addToList(String taskName) {
+        try {
+            DukeSortInputTask.dukeSortInputTask(list, taskLength, taskName);
+            taskLength += 1;
+            System.out.println("added: " + taskName);
+        } catch (DukeExceptions e) {
+            System.out.println(e.getMessage());
+        } catch (ArrayIndexOutOfBoundsException e) {
+            System.out.println("Did you forget to specify a command?");
+        }
+        System.out.println(spacer);
+    }
+
+    /** Prints a list of all tasks and its marked status to user. */
+    public static void listTasks() {
+        Integer taskIndex = 1;
+        System.out.println("Here is your list of tasks: \n");
+        for (DukeTasks task : list) {
+            System.out.println(taskIndex + ".[" + task.getTaskType() + "] " + "[" + task.getStatusIcon() + "] " + task.toString());
```

Comment:
Just another possible way to look at the same thing:
You could move the whole `".[" + task.getTaskType() + "] " + "[" + task.getStatusIcon() + "] " ` into your overriding toString method, and to simply call `taskIndex + task.toString()` here.